### PR TITLE
doc: add how to calculate blockchain and chainstate size variables to release process

### PR DESCRIPTION
**Pull request by marcoagner**
Regarding [this](https://github.com/bitcoin/bitcoin/pull/15183#issuecomment-463133734) on https://github.com/bitcoin/bitcoin/pull/15183.
Added an "Additional information" section for this which seems reasonable to me but may not be the best place for this. Also, let me know if anything else should be documented here (like more details).

### CODE REVIEW

File: doc/release-process.md [hunk 1]
```diff
@@ -312,3 +312,22 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - Create a [new GitHub release](https://github.com/bitcoin/bitcoin/releases/new) with a link to the archived release notes.
 
   - Celebrate
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
```
Comment (fanquake) [inline doc/release-process.md]: You could mention up here that the node needs to be synced, i.e `All values should be taken from a fully synced node.`, and save repeating it 4 times below.

You could also do the same for adding overhead. i.e `Add 5-10% overhead to all values.` and save repeating, plus any ambiguity.

**Review by fanquake — COMMENTED** [review]
Concept ACK

File: doc/release-process.md (hunk 1 above)
Comment (marcoagner) [reply doc/release-process.md]: That makes sense. Thanks!

**Comment by marcoagner** [comment]
Updated to 624502912536f4840eee3df07a61935a2692ae53.

File: doc/release-process.md [hunk 2]
```diff
@@ -336,3 +336,23 @@ bitcoin.org (see below for bitcoin.org update instructions).
   - Optionally twitter, reddit /r/Bitcoin, ... but this will usually sort out itself
 
   - Celebrate
+
+### Additional information
+
+#### How to calculate `m_assumed_blockchain_size` and `m_assumed_chain_state_size`
+
+Both variables are used as a guideline for how much space the user needs on their drive in total, not just strictly for the blockchain.
+Note that all values values should be taken from a **fully synced** node and have an overhead of 5-10% added on top of its base value.
```
Comment (practicalswift) [inline doc/release-process.md]: Personally I really value values but I'm not valuing values so much that I think it should be repeated repeatedly.

Comment (marcoagner) [reply doc/release-process.md]: :rofl: thanks!

**Review by laanwj — APPROVED** [review]
ACK eb4c43e49f625895670866b89bb56ca641c4eeb7

**Pull request merged**